components/Steps/EligibilityCriteria.jsx

```diff
@@ -22,59 +24,96 @@ const Step1 = props => {
+          />
+          <Select
+            {...getInputProps('eligibilityCriteria', 'typeOfBusiness')}
+            hint="Types of businesses eligible: 1) Small and micro businesses in shared offices or workspaces without their own business rates assessment (offices or workspaces in domestic dwellings are not classed as shared offices or shared workspace, to be eligible the business must be using a commercial space where an all-inclusive rent, license or similar fee is paid and occupy this space on a regular basis as their primary business premises). 2) Small and micro businesses in the retail, hospitality and leisure sector with a rateable value of between £51,000-£60,000 (inclusive) and who have therefore been unable to benefit from the existing Retail, Hospitality and Leisure Grant: the business must be a Retail, Hospitality or Leisure business identified in the Retail, Hospitality and Leisure Grant Fund. 3) Small and micro businesses whose income and turnover is directly related to the Retail, Hospitality and Leisure sector: businesses must demonstrate clearly in their application how their business is related to the Retail, Hospitality and Leisure sector. Those businesses in the Retail, Hospitality and Leisure sector are identified in the Retail, Hospitality and Leisure Grant Fund.Regular market traders with fixed building costs who do not have their own business rates assessment: 4) A regular market trader is defined as a market trader with a permanent licence who trades 5 or more days per week in the London Borough of Hackney. A fixed building cost for a market trader includes ongoing pitch fees and storage fees.  5) Bed & Breakfasts who pay Council Tax instead of business rates and who provide visitor accommodation on a per night basis including at least one meal per day (not ‘Air B&B’ style accommodation, Houses in Multiple Occupation, or other forms of short term lets),. 6)Charity properties who occupy a commercial property with a rateable value of £15,000 or less and who are in receipt of charitable business rates relief which would otherwise have been eligible for Small Business Rates Relief or Rural Rate Relief. 7)Ofsted registered nurseries (not within a domestic premises)."
+            options={[
+              'Small or micro business in shared offices or workspaces without town business rates assessment',
+              'Small or micro businesses in the retail, hospitality and leisure sector with a rateable value of between £51,000-£60,000 (inclusive)',
+              'Small or micro business whose income and turnover is directly related to the Retail, Hospitality and Leisure sector',
+              'Regular market trader with fixed building costs',
+              'Bed & Breakfast who pay Council Tax',
+              'Charity properties who occupy a commercial property with a rateable value of £15,000 or less',
+              'Ofsted registered nurseries (not within a domestic premises)'
```

Comment:
Maybe as these are so long we could explain them first in a text box and then give shortened version in the select component